apis/management.cattle.io/v3/k8s_windows_default.go

```diff
@@ -193,24 +201,18 @@ var (
 			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.11.8-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
-			CalicoCNIBinaries:  m("rancher/calico-cni:v0.0.1-nanoserver-1803"),
-			CanalCNIBinaries:   m("rancher/canal-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.11.9-rancher1-1": {
 			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.11.9-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
-			CalicoCNIBinaries:  m("rancher/calico-cni:v0.0.1-nanoserver-1803"),
```

Comment:
You can't remove images from the versions we've already released. At least for linux images, if the system images get changed for existing cluster, it will get re-provisioned. 

To be safe, please drop it only from new k8s versions. Or introduce a new k8s version if needed.